lib/record_loader/asset_shape_loader.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# RecordLoader handles automatic population and updating of database records
+# across different environments
+# @see https://rubydoc.info/github/sanger/record_loader/
+module RecordLoader
+  class AssetShapeLoader < ApplicationRecordLoader
+    config_folder 'asset_shapes'
+
+    # Creates an AssetShape record with the given name and options. If a record
+    # with the same name already exists, it is skipped. If not, a new one is
+    # created. For each size specified in the options, Map records are generated
+    # up to that size, unless they already exist.
+    #
+    # @param name [String] the name of the AssetShape record
+    # @param options [Hash] the options to be used for creating the record
+    # @param options [String] :name If provided, this will override the 'name' parameter
+    # @option options [Integer] :horizontal_ratio The horizontal ratio of the plate
+    # @option options [Integer] :vertical_ratio The vertical ratio of the plate
+    # @option options [String] :description_strategy The strategy for describing the plate
```

Comment:
I think the descriptions for horizontal_ratio, vertical_ratio and description_strategy are not useful. Either leave them out, or provide more information, or just leave a reference to the extensive docs you made in config/default_records/asset_shapes/default_records.yml ?

Reply:
I have updated 

- the option documentation with

```
    # @option options [Integer] :horizontal_ratio the numerator in the simplest form of the plate width/height ratio
    # @option options [Integer] :vertical_ratio the denominator in the simplest form of the plate width/height ratio
```

- the default_records.yml file with 

```
# record. The "horizontal_ratio" and "vertical_ratio" options are used to define
# the plate's shape. These options represent the simplest form of the ratio
# between the number of columns and rows (i.e. width / height). Specifically,
# "horizontal_ratio" corresponds to the number of columns (numerator), and
# "vertical_ratio" corresponds to the number of rows (denominator). For
# instance, a "Standard" 96-well plate has a horizontal ratio of 3 and a
# vertical ratio of 2 because it has 12 columns (1 to 12) and 8 rows (A to H).
```

I will push changes shortly.